components/org.wso2.carbon.metrics.data.service/src/test/java/org/wso2/carbon/metrics/data/service/MetricsDataServiceTest.java

```diff
@@ -105,10 +105,21 @@ public void testSources() {
         assertEquals("The source is " + SOURCE, SOURCE, sources[0]);
```

Comment:
Add more meaning full message for assets here what will happen if source[0] is null